### PR TITLE
EPMDJ-9030 Update launch parameters

### PR DESCRIPTION
**Pull request by Mazel-Tovr**
add plugin id to autotest agent

### CODE REVIEW

**Review by IliaSolovev — APPROVED** [review]

File: docs/configuration/launch-parameters.mdx [hunk 1]
```diff
@@ -137,6 +137,7 @@ Or use "DRILL_AGENT_CONFIG_PATH" parameter:
 | adminPort   | no       | 8080          |                   |                                                                     |
 | agentId     |          | ""            |                   | like in Java agent                                                  |
 | groupId     |          | ""            |                   | like in Java agent                                                  |
+| pluginId    | yes      | ""            | test2code         | the id of the plugin to which the auto test agent will send requests, you need to specify only if you use the command line to start the autotest agent, using the gradle/maven plugin the value will be set automatically |
```
Comment (LikhanovMaxim) [inline docs/configuration/launch-parameters.mdx]: create task : default value=test2code for command line
```suggestion
| pluginId    | yes      | test2code    |  | the id of the plugin to which the auto test agent will send requests |
```

**Review by LikhanovMaxim — CHANGES_REQUESTED** [review]

**Review by RomanAchilov — CHANGES_REQUESTED** [review]
Nice branch and commit name, where task id?

**Comment by Mazel-Tovr** [comment]
> Nice branch and commit name, where task id?

@RomanAchilov  look at commit name, there is a task id there. I don't see a problem that there is no task name in the name of the pr, because it is only on the GitHub site. It could be fixed in 2 clicks.
And yeah, I agree with u, the branch name is really cool.

**Comment by RomanDavlyatshin** [comment]
obsolete

**Pull request closed without merging**